src/containers/dashboard/MapHead.css

```diff
@@ -60,9 +80,9 @@ input[type="range"] {
 
 input[type="range"]::-webkit-slider-thumb {
   -webkit-appearance: none;
-  height: 20px;
-  width: 5px;
-  border-radius: 8px;
+  height: 30px;
```

Comment:
Let's use rem in the future where we can

Reply:
Bad habit. I honestly never knew the difference between using px or em and rem until now.